unified-codes/libs/ui/src/lib/components/entity/organisms/EntityBrowser.tsx

```diff
@@ -1,18 +1,21 @@
 import * as React from 'react';
 
 import Grid from '../../layout/atoms/Grid';
+import { Backdrop, CircularProgress } from '../../feedback';
 
 export interface IEntityBrowserClasses {
   root?: string;
-  toggleBarContainer?: string,
-  searchBarContainer?: string,
-  tableContainer?: string,
+  backdrop?: string;
+  searchBarContainer?: string;
+  tableContainer?: string;
+  toggleBarContainer?: string;
 }
 
 export interface IEntityBrowserProps {
   classes?: IEntityBrowserClasses;
   onMount: () => void;
   onUnmount: () => void;
+  loading?: boolean;
```

Comment:
Minor gripe here from TS as it is `boolean` in the Backdrop component (rather than `boolean | undefined`)

Reply:
thanks @katherine-sussol! I saw that at one point and didn't fix it up. 
Done now 😄 
( offroad: have also, finally, removed the horizontal scrollbar )